src/style.css

```diff
@@ -0,0 +1,68 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts\Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts\Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+body {
+  margin-top: 0;
+}
+
+.search_bar {
+  background: rgba(255, 255, 255, 1);
+  background-image: url(images/Search.svg);
```

Comment:
remove background property, it conflicts with background-image
and by default background color will be white